src/main/java/liqp/RenderTransformer.java

```diff
@@ -0,0 +1,93 @@
+package liqp;
+
+import java.io.Writer;
+
+/**
+ * <p>
```

Comment:
@msangel see this javadoc explanation. Especially note the "7x / 35x smaller" part :)